includes/class-creativecommons.php

```diff
@@ -1125,10 +1125,11 @@ public function license_html_rdfa( $deed_url, $license_name, $image_url,
 									$title_work, $is_singular, $attribute_url,
 									$attribute_text, $source_work_url,
 									$extra_permissions_url, $additional_attribution_txt, $title, $title_url, $author, $author_url ) {
-		list($img_width, $img_height) = getimagesize( $image_url );
-		$lazy = function_exists('wp_lazy_loading_enabled') && wp_lazy_loading_enabled('img', 'license_html_rdfa');
+		$image_path = str_replace( CCPLUGIN__URL, CCPLUGIN__DIR, $image_url );
```

Comment:
I believe the two global variables here should be properly escaped. the value of $image_url is already escaped in its definition further up in the file, but the use of these two globals here doesn't provide them as escaped values.